Start time: 06:55
dataset: spacev10m
N: 10000000
dataset file: /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs
query file: /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs
attr file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json
query range file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json
ground truth file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json
K: 10
nprobe: 150
collection name: collection_spacev10m_sel_1_500_random_100
2025-04-09 21:56:01,910 [ERROR][_create_connection]: Failed to create new connection using: af7b75d6c80a4c9bbc522e1805e99a62 (milvus_client.py:923)
Traceback (most recent call last):
  File "utils/pymilvus_ivfpq.py", line 67, in <module>
    client = MilvusClient(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 67, in __init__
    self._using = self._create_connection(
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 924, in _create_connection
    raise ex from ex
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/milvus_client/milvus_client.py", line 921, in _create_connection
    connections.connect(using, user, password, db_name, token, uri=uri, **kwargs)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 461, in connect
    connect_milvus(**kwargs, user=user, password=password, token=token, db_name=db_name)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/orm/connections.py", line 411, in connect_milvus
    gh._wait_for_channel_ready(timeout=timeout)
  File "/home/mocheng/anaconda3/envs/py38/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 155, in _wait_for_channel_ready
    raise MilvusException(
pymilvus.exceptions.MilvusException: <MilvusException: (code=2, message=Fail connecting to server on localhost:19530, illegal connection params or server unavailable)>
Command exited with non-zero status 1
	Command being timed: "python -u utils/pymilvus_ivfpq.py spacev10m 10000000 /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json 10 150 collection_spacev10m_sel_1_500_random_100 query 100 100"
	User time (seconds): 1.93
	System time (seconds): 0.11
	Percent of CPU this job got: 19%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:10.69
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 76800
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 27
	Minor (reclaiming a frame) page faults: 22194
	Voluntary context switches: 2028
	Involuntary context switches: 700
	Swaps: 0
	File system inputs: 3784
	File system outputs: 0
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
Start time: 06:56
dataset: spacev10m
N: 10000000
dataset file: /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs
query file: /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs
attr file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json
query range file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json
ground truth file: /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json
K: 10
nprobe: 150
collection name: collection_spacev10m_sel_1_500_random_100
collection  collection_spacev10m_sel_1_500_random_100  exists
collection size: data: ["{'count(*)': 10000000}"]
query d: 100
get query szie: 10000
get range cnt: 10000
ngt= 10000  nq= 10000  k= 10
ids shape: 10000 10
gt shape: (10000, 10)
milvus get  86927  postive res from  100000  results, recall@ 10 : 0.86927
recall dist(10\% per bucket):  [2.000e-04 1.100e-03 2.500e-03 4.800e-03 1.190e-02 2.550e-02 4.430e-02
 8.770e-02 1.506e-01 3.014e-01 3.700e-01]
10000  queries cost: 92.53378129005432 , qps: 108.06864110150458
	Command being timed: "python -u utils/pymilvus_ivfpq.py spacev10m 10000000 /mnt/data/mocheng/dataset/spacev10m/base10M.fvecs /mnt/data/mocheng/dataset/spacev10m/query10k.fvecs /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/attr_sel_1_500_random.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/qrangesel_1_1_500_random_20.json /mnt/data/mocheng/dataset/spacev10m/label/sel_1_500_random/sif_gt_sel_1_1_500_random_20_10.json 10 150 collection_spacev10m_sel_1_500_random_100 query 100 100"
	User time (seconds): 21.71
	System time (seconds): 2.68
	Percent of CPU this job got: 20%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 1:59.87
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 91864
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 20
	Minor (reclaiming a frame) page faults: 65891
	Voluntary context switches: 24245
	Involuntary context switches: 501910
	Swaps: 0
	File system inputs: 496
	File system outputs: 24
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
